src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md

```diff
@@ -127,10 +127,19 @@ foreach($uami in $uamis){
 Update-AzSynapseWorkspace -Name ContosoWorkspace -UserAssignedIdentityAction Set -UserAssignedIdentityId $uamilist
 ```
 
-This commands updates workspace with user assigned managed identites $uamilist that will cover current identities.
+
+This commands set the transparent data encryption protector for a workspace, and it is using System Assigned Managed Identity to access the Azure Key Vault.
```

Comment:
This sentence you removed/updated is actually for Example 8. Can you please revert the change?

Reply:
> This sentence you removed/updated is actually for Example 8. Can you please revert the change?

sorry, yes, you are right, I have reverted the change.